opentelemetry-api/src/opentelemetry/trace/span.py

```diff
@@ -77,7 +77,7 @@ def is_recording(self) -> bool:
     @abc.abstractmethod
     def set_status(self, status: Status) -> None:
         """Sets the Status of the Span. If used, this will override the default
-        Span status, which is OK.
+        Span status, which is UNSET.
```

Comment:
Should we remove the "which is ..." part? Some implementations of span might have different defaults.

Reply:
Makes sense to remove that if some implementations have different defaults. I was reading through API and implementation details and found a disparity with [sdk](https://github.com/open-telemetry/opentelemetry-python/blob/8cc92600158aba9d9de272f66a29a8422e3e5a66/opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py#L441). Small question for my own understanding, when will there be different defaults?

Reply:
It is up to the implementor. Perhaps they want their own implementation to default to `Ok`, where as the OT default SDK defaults to `UNSET`.